main.go

```diff
@@ -204,6 +357,52 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 
 		ch <- prometheus.MustNewConstMetric(e.evictions, prometheus.CounterValue, parse(s, "evictions"))
 		ch <- prometheus.MustNewConstMetric(e.reclaimed, prometheus.CounterValue, parse(s, "reclaimed"))
+
+		ch <- prometheus.MustNewConstMetric(e.active_slabs, prometheus.GaugeValue, parse(s, "active_slabs"))
+		ch <- prometheus.MustNewConstMetric(e.malloced, prometheus.GaugeValue, parse(s, "total_malloced"))
+
+		for slab, u := range t.Items {
+			slab := strconv.Itoa(slab)
+			ch <- prometheus.MustNewConstMetric(e.itemsNumber, prometheus.CounterValue, parse(u, "number"), slab)
+			ch <- prometheus.MustNewConstMetric(e.itemsAge, prometheus.CounterValue, parse(u, "age"), slab)
+			ch <- prometheus.MustNewConstMetric(e.itemsEvicted, prometheus.GaugeValue, parse(u, "evicted"), slab)
```

Comment:
Check the types of all of these metrics, they seem to be all counters and not gauges (e.g. they'll never go down). A counter should always have the `_total` suffix, a gauge should never have the `_total` suffix.